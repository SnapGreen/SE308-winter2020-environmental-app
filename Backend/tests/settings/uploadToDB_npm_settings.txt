settings check:
	SETTINGS: Backend/scripts/settings_npm.txt
	DATADIR: Backend/data/
	FDADIR: FDA/
	FDADATADIR: Backend/data/FDA/
	SUFFIX_LEN: 4
	PRODS_PER_JSON: 250
	FB_WRITES_PER_DAY: 16500
	TESTDIR: :Backend/tests/
	TESTSETTINGSDIR: :Backend/tests/settings/
	TEST_POPDB_SETTINGS: :Backend/tests/settings/populateDB_settings.txt
	TEST_UPLOADTODB_SETTINGS: :Backend/tests/settings/uploadToDB_settings.txt
	SPLIT_PREFIX: branded_food_
	OUTFILE_END: .json
	UPLOAD_SLEEP: 1
	LOGDIR: Backend/logs/
	UPLOADLOGDIR: Backend/logs/uploads/
	SERVER_POPULATED: false
	DONE_UPLOADING: false
	USAGE:
		Usage: ./uploadToDB.sh [OPTIONS] (-h for help)
	HELP:
		Usage: ./uploadToDB.sh [OPTIONS] (-h for help)
		OPTIONS:
			-b: upload without prompt
			-s: print settings only
			-h: print help
