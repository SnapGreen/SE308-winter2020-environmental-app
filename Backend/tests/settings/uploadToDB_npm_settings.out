settings check:
	SETTINGS: Backend/scripts/settings_npm.txt
	DATADIR: 
	FDADIR: 
	FDADATADIR: 
	SUFFIX_LEN: 
	PRODS_PER_JSON: 
	FB_WRITES_PER_DAY: 
	TESTDIR: 
	TESTSETTINGSDIR: settings/
	TEST_POPDB_SETTINGS: settings/populateDB_settings.txt
	TEST_UPLOADTODB_SETTINGS: settings/uploadToDB_settings.txt
	SPLIT_PREFIX: 
	OUTFILE_END: 
	UPLOAD_SLEEP: 
	LOGDIR: 
	UPLOADLOGDIR: uploads/
	SERVER_POPULATED: 
	DONE_UPLOADING: 
	USAGE:
		Usage: ./uploadToDB.sh [OPTIONS] (-h for help)
	HELP:
		Usage: ./uploadToDB.sh [OPTIONS] (-h for help)
		OPTIONS:
			-b: upload without prompt
			-n: test with settings relative to repo root
			-ns: print settings relative to repo root only
			-s: print settings only
			-t: test mode (silent)
			-h: print help
